giftless/storage/google_cloud.py

```diff
@@ -60,11 +66,11 @@ def get_size(self, prefix: str, oid: str) -> int:
 
     def get_upload_action(self, prefix: str, oid: str, size: int, expires_in: int,
                           extra: Optional[Dict[str, Any]] = None) -> Dict[str, Any]:
-        filename = extra.get('filename') if extra else None
```

Comment:
It would be good to maintain support for the `x-filename` "extra" property. I know it's somewhat poorly documented, but this allows us (at least with the Azure backend) to specify a value for the `Content-Disposition` header, which enables to set the file name in the browser when downloading (otherwise the name of the file suggested in the "Save As" dialog in the browser is the sha256 which is not pretty). 

I'm not sure how to obtain this in GCS, but maybe there is a away.

Reply:
Is it worth addressing this in this same PR? Wouldnt it be better to open another PR?

Reply:
Yes you are right. I just noticed it because it was removed.